notification.go

```diff
@@ -85,7 +90,11 @@ func (p *Payload) SetCustomValue(key string, value interface{}) error {
 }
 
 func (p *Payload) MarshalJSON() ([]byte, error) {
-	p.customValues["aps"] = p.APS
+	if len(p.MDM) != 0 {
```

Comment:
I can look into that suggestion, but I actually need "aps" to not be present if it's completely empty.